openid-federation-1_0.xml

```diff
@@ -3523,70 +3523,64 @@
 	    To validate a Trust Mark Issuer, follow the procedure
 	    defined in <xref target="resolving_trust"/>.
-	    To validate a Trust Mark, either:
-            <list style="symbols">
-              <t>
-                Validate the signature of the Trust Mark JWT and verify that
-                it has not expired.
-              </t>
-              <t>
-                If the Trust Mark type identifier appears in the
-                <spanx style="verb">trust_mark_owners</spanx> claim
-		of the Trust Anchor's Entity Configuration,
-                verify that the Trust Mark contains
-                a <spanx style="verb">delegation</spanx>
-                claim.
-                The claims for the Trust Mark type identifier
-                in the <spanx style="verb">trust_mark_owners</spanx> value
-                are used in the following way:
-                <list style="symbols">
-                  <t>
-                    The signature of the <spanx style="verb">delegation</spanx> JWT
-                    MUST verify with a key from <spanx style="verb">jwks</spanx> claim.
-                  </t>
-                  <t>
-                    The issuer of the <spanx style="verb">delegation</spanx> JWT
-		    MUST match the <spanx style="verb">sub</spanx> value
-		    in this set of claims.
-                  </t>
-                </list>
-              </t>
-	    </list>
-	  or:
-            <list style="symbols">
-              <t>
-                Use the Trust Mark Issuer status endpoint to verify that the
-                Trust Mark is still active
-                as described in <xref target="status_endpoint"/>.
+          </t>
+      <t>
+        Henceforth instance is used as a shorthand for trust mark instance. An
+        instance is a signed JWT.
+        To validate an instance the following has to be done (if any of these
+        tests fail then the validation fails):
+        <list style="numbers">
+          <t>
+            Verify that the value of the instance’s
+            <spanx style="verb">trust_mark_id</spanx>  is listed in
+            the TA’s <spanx style="verb">trust_mark_issuers</spanx> and if so
+            that the instance's <spanx style="verb">iss</spanx> appears
+            in the corresponding list of entity ids. Note that the list may be
+            an empty list which signifies that anyone can issue a trust mark
+            with the <spanx style="verb">id</spanx> in question.
+          </t>
+          <t>
+            Validate the signature of the instance and verify that it has not
+            expired and/or send a query to the Trust Mark issuer’s Trust Mark
+            Status endpoint. The Trust Mark Status endpoint will not only
+            check if the instance has been issued by itself but also if is
+            still valid (not timed out) and whether it has been revoked or not.
+          </t>
+          <t>
+            If the <spanx style="verb">trust_mark_id</spanx> of the instance
```

Comment:
Hint for trust_mark_id > trust_mark_type change: https://github.com/openid/federation/pull/219